src/orchestrator-kubernetes/src/lib.rs

```diff
@@ -975,6 +975,21 @@ impl NamespacedOrchestrator for NamespacedKubernetesOrchestrator {
             .context("`image` is not ORG/NAME:VERSION")?
             .to_string();
 
+        let container_security_context = Some(SecurityContext {
+            privileged: Some(false),
+            run_as_non_root: Some(true),
+            allow_privilege_escalation: Some(false),
+            seccomp_profile: Some(SeccompProfile {
+                type_: "RuntimeDefault".to_string(),
+                ..Default::default()
+            }),
+            capabilities: Some(Capabilities {
+                drop: Some(vec!["ALL".to_string()]),
```

Comment:
I think I'm a little surprised we can drop all capabilities for clusters. We don't need any to run `rocksdb`/`lgalloc`/establish source (with `ssh`) connections?

Reply:
[Here's a list of the capabilities that could be allowed/are currently allowed](https://man7.org/linux/man-pages/man7/capabilities.7.html) last time I checked we're not doing any of these.


There is a note about need `CAP_SYS_ADMIN` to do `MADVISE_HWPOISION` (giving that cap would defeat the purpose of restricting the caps in the first place of course) however looking at [the lgalloc code](https://github.com/antiguru/rust-lgalloc/blob/15080624f954f2bf2d38e83158bddf8a790f29b5/src/lib.rs#L101) we don't use that system call, so it should be fine. I would of course like to have a test for lgalloc if we don't already.